client/js/components/content-slider.js

```diff
@@ -365,11 +365,17 @@ const contentSlider = (el, options) => {
     }, data);
   }
 
+  function blurCurrentTarget({ currentTarget }) {
+    currentTarget.blur();
+  }
+
   setup();
 
   // Handle click
   sliderElements.prevControl.addEventListener('click', prevSlide, true);
+  sliderElements.prevControl.addEventListener('mouseup', blurCurrentTarget);
   sliderElements.nextControl.addEventListener('click', nextSlide, true);
+  sliderElements.nextControl.addEventListener('mouseup', blurCurrentTarget);
```

Comment:
The `click` event fires on pressing enter or space, but we don't want to blur the control buttons if the user is navigating with the keyboard. The `mouseup` event fires after `click`, and (obviously) not relative to any keyboard events. This isn't clear – I'll add a comment.